src/xscen/catutils.py

```diff
@@ -223,7 +229,7 @@ def _name_parser(
         return None
 
     d["path"] = abs_path
-    d["format"] = path.suffix[1:]
+    d["format"] = SUFFIX_TO_FORMAT.get(path.suffix)
```

Comment:
Is `d["format"] = None` possible/covered?

Reply:
hmm I guess not, I'll put the previous value as default.